src/AltinnCore/Designer/gulpfile.js

```diff
@@ -23,84 +23,86 @@ const jsServDevModuleFile3 = '../../react-apps/applications/service-development/
 const jsServDevMonacoWorker1 = '../../react-apps/applications/service-development/js/react/editor.worker.js';
```

Comment:
this file must not be changed

Reply:
this file is restored to the master version

Reply:
Agreed! 👍